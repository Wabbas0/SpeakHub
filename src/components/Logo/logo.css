.logo-container {
  display: flex;
  align-items: center;
}

.logo-container__image {
  width: 42px;
  margin: 5px;
}

.logo-container__text {
  margin-bottom: 0;
  font-weight: bold;
  font-size: 1.4em;
}

#logo-container span {
  color: #fa4b34;
}
