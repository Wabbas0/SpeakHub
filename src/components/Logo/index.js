// @flow
import React from "react";
import "./logo.css";

const Logo = ({ url }: { url: string }) => (
  <div className="logo-container">
    <img className="logo-container__image" src={url} alt="application logo" />
    <h1 className="logo-container__text">
      Speak<span>Hub</span>
    </h1>
  </div>
);

export default Logo;
