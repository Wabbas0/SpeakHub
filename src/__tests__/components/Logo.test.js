import React from "react";
import { create } from "react-test-renderer";
import Logo from "../../components/Logo";

describe("Logo component", () => {
  test("it matches the snapshot when passing speakap logo", () => {
    const component = create(
      <Logo url="https://www.speakap.com/hubfs/favicon.png" />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
