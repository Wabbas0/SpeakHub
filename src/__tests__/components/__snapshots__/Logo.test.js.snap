// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component it matches the snapshot when passing speakap logo 1`] = `
<div
  className="logo-container"
  id="logo-container"
>
  <img
    alt="application logo"
    className="logo-container__image"
    src="https://www.speakap.com/hubfs/favicon.png"
  />
  <h1
    className="logo-container__text"
  >
    Speak
    <span>
      Hub
    </span>
  </h1>
</div>
`;
